charts/dependency-track/templates/api-server/deployment.yaml

```diff
@@ -0,0 +1,148 @@
+{{- if eq .Values.apiServer.deploymentType "Deployment" }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "dependencytrack.apiServerFullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "dependencytrack.apiServerLabels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "dependencytrack.apiServerSelectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels: {{- include "dependencytrack.apiServerSelectorLabels" . | nindent 8 }}
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path: /metrics
+        {{- with .Values.apiServer.annotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.common.image.pullSecrets }}
+      imagePullSecrets: {{- toYaml . | nindent 6 }}
+      {{- end }}
+      initContainers:
+      {{- if .Values.frontend.initContainers }}
+      {{- toYaml .Values.frontend.initContainers | nindent 6 }}
+      {{- end }}
+      serviceAccount: {{ include "dependencytrack.serviceAccountName" . }}
+      containers:
+      - name: {{ include "dependencytrack.apiServerName" . }}
+        image: {{ include "dependencytrack.apiServerImage" . }}
+        imagePullPolicy: {{ .Values.apiServer.image.pullPolicy }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          runAsNonRoot: true
+          readOnlyRootFilesystem: true
+          seccompProfile:
+            type: RuntimeDefault
+        {{- with .Values.apiServer.command }}
+        command: {{ toYaml . | nindent 8 }}
+        {{- end }}
+        {{- with .Values.apiServer.args }}
+        args: {{ toYaml . | nindent 8 }}
+        {{- end }}
+        resources: {{- toYaml .Values.apiServer.resources | nindent 10 }}
+        env:
+        - name: ALPINE_DATA_DIRECTORY
+          value: "/data"
+        - name: ALPINE_METRICS_ENABLED
+          value: "true"
+        {{- if .Values.postgresql.enabled }}
+        - name: ALPINE_DATABASE_MODE
+          value: "external"
+        - name: ALPINE_DATABASE_DRIVER
+          value: "org.postgresql.Driver"
+        - name: ALPINE_DATABASE_URL
+          value: jdbc:postgresql://{{ .Release.Name }}-postgresql/{{ .Values.postgresql.postgresqlDatabase }}
+        - name: ALPINE_DATABASE_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Release.Name }}-postgresql
+              key: postgresql-password
+        - name: ALPINE_DATABASE_USERNAME
+          value: {{ .Values.postgresql.postgresqlUsername }}
+        {{- end }}
```

Comment:
I think this part doesn't belong here, there is no bundled PostgreSQL database.

Reply:
fixed in [120e7cb](https://github.com/DependencyTrack/helm-charts/pull/102/commits/120e7cb6edc73952e951fcb41e0de10bb54935c5)